### PR TITLE
Add changelog entry for Release 8

### PR DESCRIPTION
**Pull request by StachuDotNet**
No changelog

This also corrects the heading level of some headers in Release 7.

### CODE REVIEW

**Comment by StachuDotNet** [comment]
Just realized - I'd generated the draft of this changelog before November actually finished - darklang/dark#4617 needs to be added still. :construction:

**Comment by pbiggar** [comment]
Changed some wording - please review and merge when ready!

File: docs/reference/changelog.md [hunk 1]
```diff
@@ -18,7 +18,8 @@ sidebar_label: Changelog
 - Correct documentation of how code is formatted via terminal
   ([#294](https://github.com/darklang/docs/pull/294))
 
-- Fix some links to code ([#293](https://github.com/darklang/docs/pull/293))
+- Fix some links to Darklang code
```
Comment (StachuDotNet) [inline docs/reference/changelog.md:21]: I was torn on this. "Darklang code" Feels like it implies code written in Darklang a bit. Probably fine though

Comment (pbiggar) [reply docs/reference/changelog.md:21]: Yeah, this is a good point.

**Pull request merged**